app/src/test/java/com/example/mylittlelibrary/BookUnitTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.mylittlelibrary
+
+import com.example.mylittlelibrary.data.Book
+import org.junit.Assert
+import org.junit.Test
+import java.util.*
+
+class BookUnitTest {
+    @Test
+    fun `given a book dto when name is MyLittleLibrary and lendTo is MyLittleLibraryI then code is MyLittleLibrary and name is MyLittleLibraryI`() {
+        var country = Book("MyLittleLibrary", "MyLittleLibraryI", Date())
```

Comment:
Add unit test for the project. URL: https://developer.android.com/training/testing/fundamentals